src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -436,6 +437,19 @@ public bool RunEnabled
                 RaisePropertyChanged("RunEnabled");
             }
         }
+
+        private bool showRunPreview;
+
+        public bool ShowRunPreview
```

Comment:
Similar to above, this property should live in the view model. It doesn't make sense in the model.